server/integration/testsuite/pom.xml

```diff
@@ -1245,10 +1245,10 @@
                                 <configuration>
                                     <skip>${skipTests}</skip>
                                     <target>
-                                        <ant antfile="${basedir}/src/test/resources/ant/infinispan-server.xml" target="kill-server">
+                                        <ant antfile="../src/main/ant/infinispan-server.xml" target="kill-server">
```

Comment:
I'd prefer to use variables rather than relative paths, what was the problem?

Reply:
The problem was that `src/test/resources/ant/infinispan-server.xml` no longer exists, `infinispan-server.xml` is now in the parent module/directory.

I can add `${basedir}` in front if you prefer, I just copied the path that we were already using elsewhere.